api_demo/models.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import (Column, ForeignKey,
+                        Integer, String, DateTime, Text
+                        )
+
+from sqlalchemy.ext.declarative import declarative_base
+from datetime import datetime
+
+Base = declarative_base()
+
+
+class BlogBase(object):
```

Comment:
Please do
```suggestion
class BlogBase:
```
All classes inherit object in python, you don't have to inherit them explicitly

Reply:
Even better do it this way
```suggestion
class BaseModel(Base):
```
And remove pub_date from this class. Why do we need this field for Category? You may create smth like CreatedMixin and put there created_at field